### ISSUE TITLE
[BUG] KeyError: 'profile_def' on search

### ISSUE THREAD

**Issue by spurll** [ScrappyCocco/HowLongToBeat-PythonAPI#29]
**Describe the bug**
When searching by name or by ID, `KeyError: 'profile_dev'` occurs

**To Reproduce**
```
Python 3.12.3 (main, Jul 31 2024, 17:43:48) [GCC 13.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from howlongtobeatpy import HowLongToBeat
>>> HowLongToBeat()
<howlongtobeatpy.HowLongToBeat.HowLongToBeat object at 0x7b827079a5d0>
>>> HowLongToBeat().search("Ogre Battle")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/goodplays/venv/lib/python3.12/site-packages/howlongtobeatpy/HowLongToBeat.py", line 65, in search
    return self.__parse_web_result(game_name, html_result, None, similarity_case_sensitive)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/goodplays/venv/lib/python3.12/site-packages/howlongtobeatpy/HowLongToBeat.py", line 134, in __parse_web_result
    parser.parse_json_result(html_result)
  File "/opt/goodplays/venv/lib/python3.12/site-packages/howlongtobeatpy/JSONResultParser.py", line 40, in parse_json_result
    new_game_entry = self.parse_json_element(game)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/goodplays/venv/lib/python3.12/site-packages/howlongtobeatpy/JSONResultParser.py", line 61, in parse_json_element
    current_entry.profile_dev = input_game_element["profile_dev"]
                                ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
KeyError: 'profile_dev'
```

**HLTB API Version**
v1.0.7

**Comment by tlt21** [comment]
Looks like "profile_platform" is also missing. Had to comment out both to get search to work. 
![image](https://github.com/user-attachments/assets/dceba6e2-0789-4498-ba3f-c5b3cbb96ae0)

**Comment by ScrappyCocco** [comment]
It looks like the JSON no longer has those values
Example json:
```
{
   "game_id":42818,
   "game_name":"Celeste",
   "game_name_date":0,
   "game_alias":"",
   "game_type":"game",
   "game_image":"42818_Celeste.jpg",
   "comp_lvl_combine":0,
   "comp_lvl_sp":1,
   "comp_lvl_co":0,
   "comp_lvl_mp":0,
   "comp_main":29770,
   "comp_plus":51483,
   "comp_100":135404,
   "comp_all":44161,
   "comp_main_count":3039,
   "comp_plus_count":1679,
   "comp_100_count":657,
   "comp_all_count":5375,
   "invested_co":0,
   "invested_mp":0,
   "invested_co_count":0,
   "invested_mp_count":0,
   "count_comp":10830,
   "count_speedrun":98,
   "count_backlog":8912,
   "count_review":3795,
   "review_score":88,
   "count_playing":335,
   "count_retired":615,
   "profile_popular":1263,
   "release_world":2018
}
```
Should I put them to None if not present?

**Comment by ScrappyCocco** [comment]
I was thinking about doing
```python
if "profile_dev" in input_game_element:
    current_entry.profile_dev = input_game_element["profile_dev"]
if "profile_platform" in input_game_element:
    current_entry.profile_platforms = input_game_element["profile_platform"].split(", ")
if "release_world" in input_game_element:
    current_entry.release_world = input_game_element["release_world"]
```
So if they ever appear again they're gonna be back there, otherwise they are None

**Comment by spurll** [comment]
Seems reasonable to me, though wouldn't it be a bit more Pythonic to just `get` them, skipping the `if`?

```
current_entry.profile_dev = input_game_element.get("profile_dev")
```

I might be wrong, I've been pretty much out of the software game for a few years.

**Comment by ScrappyCocco** [comment]
Yes you're right, I'll do it this way, sorry I don't use Python very much

**Comment by ScrappyCocco** [comment]
Fixed on version 1.0.8
